acct-user/spectrum/spectrum-0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the Spectrum messaging transport"
+
+ACCT_USER_GROUPS=( "spectrum" )
+ACCT_USER_HOME="/var/lib/spectrum2"
+ACCT_USER_HOME_OWNER="spectrum:spectrum"
```

Comment:
I see (and highly appreciate!) that you have shown the diligence in following the process, by submitting this to the gentoo-dev maillist.
`mjo` has pointed that this line is not necessary (in his review of acct-user/minecraft), and you have applied that to v2 of your patch which you have also submitted to the maillist.

But this is apparently the v1 commit.

No big deal, I can fix this up while merging it. All the contents look legitimate. Could you please just double-check that there's no other obsoleteness in this submission?

Reply:
Thanks for the catch. I just now checked all of my acct-* PRs and it seems, somehow I must only here forgotten to update the patch here. Fixed PR.